### PR TITLE
[TEST PR] - added donate landing and help page to percy list

### PR DESCRIPTION
**Pull request by danielfmiranda**
This PR was used to trouble shoot issues that we were having with percy not showing recaptcha or fundraise up forms.

Due to the troubleshooting of the percy issue, github experiencing issues with PR merging, and rebasing to `2023-eoy` on account of the code freeze, this PR has become quite messy and hard to follow.

So I have created https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11595 to house the actual fix in its place.

### CODE REVIEW

**Comment by danielfmiranda** [comment]
Hi @mmmavis! I noticed a slight difference between the percy screenshots that I am finding on this PR and the ones that are being generated by Percy when running it on my local machine and was hoping to hear your opinion.

When taking a look at the screenshots generated by percy on my local machine, I can see that the FundraiseUp form on the DonateLandingPage is rendered as expected:

**Donate Landing Page (Local Percy test):**

Screenshot 2023-12-01 at 12 07 12 AM


However, if I click on the percy results on this PR, I see that it is not being rendered.



**Donate Landing Page (PR Percy test):**

Screenshot 2023-12-01 at 00-07-55 Build #15425 - Mozilla Foundation - Percy Visual testing as a service


Originally, I had thought that this could be due to some missing CSP directives on the percy side. However after comparing I dont seem to find any differences. 

I was wondering, could it be because this form is rendered in by an external script? And if so, is this something that the team has experienced before?


Thanks in advance for the help!

**Review by mmmavis — COMMENTED** [review]
Hey @danielfmiranda sorry for getting back to you on PR late. This has slipped through the cracks on my end. Did you try updating CSP in `visual-regression-testing.yml`?
